requests_toolbelt/adapters/appengine.py

```diff
@@ -27,6 +31,7 @@ class AppEngineAdapter(adapters.HTTPAdapter):
         >>> from requests_toolbelt.adapters import appengine
         >>> s = requests.Session()
         >>> if using_appengine():
```

Comment:
I've reworked the comment to use is_appengine_sandbox(), which in addition to working code, also handles Managed VMs properly.